### ISSUE TITLE
can not analyze memory leak

### ISSUE THREAD

**Issue by darkgirl** [ssbssa/heob#16]
Yesterday, i want to use heob to check my qt application. but, when i run heob for12 hours, it still running and the Cpu occupancy rate is still very high, the memory used 4G but almost no decline.
I don't know if it is running normally.

my qt version is 5.11
and my heob version is 2.1.7
my vs version is 2017 vc++ 15.0 x64

![heob](https://user-images.githubusercontent.com/28688741/48524012-8f906980-e8b9-11e8-84b5-f1b3036907ad.PNG)

Thanks for your participation.

**Comment by ssbssa** [comment]
I'm not really sure what you're asking me here.
So you mean your program is running for 12 hours, or does it never stop?

Please explain your situation a bit more.

**Comment by darkgirl** [comment]
That day, I found that a program memory is very large, but I didn't find a memory leak in the code, so I want to use the detection software to see what went wrong. But on Linux I used Valgrind that doesn't seem to run on windows. So I found the heob. But when I ran the heob according to the settings on the diagram, it seems to have detected the leaked memory and has been analyzing it, but it was not analyzed until the evening, so I didn't turn off the computer and let it keep running. The next day I found out that it is still running, so I would like to ask if there is a problem with my settings.I am sorry to reply now, because I found no one to reply to me, so I continued to find the problem myself. I found out that there was a memory leak in the algorithm of others. I happened to check my GitHub and saw your reply. Thanks to your patiently reply, and thanks to google translate.
